include/mesh.tcc

```diff
@@ -1895,6 +1895,8 @@ void mpm::Mesh<Tdim>::create_nodal_properties() {
                                        materials_.size());
     nodal_properties_->create_property("momenta", nodes_.size() * Tdim,
                                        materials_.size());
+    nodal_properties_->create_property("change_in_momenta",
```

Comment:
Could we store `nodes_.size() * Tdim` as a `const unsigned` variable in this function?